src/libaktualizr/storage/invstorage.h

```diff
@@ -87,7 +87,7 @@ class StorageTargetRHandle {
     file.close();
   }
 
-  // FIXME this function loads the whole image to the memory
+  // TODO: this function loads the whole image to the memory
```

Comment:
This comment doesn't look accurate actually, this function works buffers in blocks of 256 bytes. Or did we miss something (the comment is more recent than the code!)?

But I think @mike-sul mentioned that we still have places where this could happen, if that's the case we should also have a ticket for it and solve it quite soon.

Reply:
Good point. I'm removing this TODO. If anyone finds somewhere else where the supposed problem is occurring, please leave a comment and/or make a ticket.